app/Models/MapGroup.php

```diff
@@ -25,7 +26,17 @@
  */
 class MapGroup extends Model
 {
-    use VisibilityIDTrait, Blameable, Paginatable;
+    use VisibilityIDTrait, Blameable, Paginatable, SortableTrait;
+
+    protected $sortable = [
```

Comment:
I think only one of these is needed. Either $sortable or $sortableColumns, but not both :)